_local.tf

```diff
@@ -9,23 +9,23 @@ locals {
   registry_username = data.aws_ecr_authorization_token.this.user_name
   registry_password = data.aws_ecr_authorization_token.this.password
   dibbs_repo        = var.dibbs_repo
-  database_url = var.database_type == "postgresql" ? {
+  database_url = var.secrets_manager_postgresql_connection_string_version != "" ? {
```

Comment:
should we rename the variable to `secrets_manager_database_connection_string_version` or similar? since it isn't used only for postgres anymore